pkg/apis/scheduling/v1alpha2/register.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	AddToScheme   = SchemeBuilder.AddToScheme
+)
+
+const (
+	// GroupName is the group name used in this package.
+	GroupName = "scheduling.sigs.dev"
```

Comment:
I am not sure why changed the group name, in v1alpha1 it is called "scheduling.incubator.k8s.io"

Reply:
Group name has been changed to v1alpha2 only. As suggested by @k82cn

Reply:
`v1alpha2 ` is version, is there any consideration i missed for groupname change?

Reply:
This was the issue https://github.com/kubernetes-sigs/kube-batch/issues/815 in kube-batch in which groupname was specified by @k82cn. So have followed the same here too.

Reply:
let's use new groupname.

Reply:
@k82cn @hzxuzhonghu Please suggest new one.

Reply:
> @k82cn @hzxuzhonghu Please suggest new one.

The new one is  kubernetes-sigs/kube-batch#815 :)

Reply:
@k82cn I think you mean `scheduling.sigs.k8s.io`, so I will make changes accordingly

Reply:
FYI: 
We already have `"batch.volcano.sh"`  `"bus.volcano.sh"` `"scheduling.incubator.k8s.io"`.

Personally, i prefer `scheduling.volcano.sh` if there is no other consideration.

Reply:
Ok, talked in private. We should keep it consistent with `kube-batch`.

Reply:
We used to promise kubeflow to follow up kube-batch's api, so let's just cherry-pick this from kube-batch :)